<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleSheetConfig</class>
 <widget class="QWidget" name="ScheduleSheetConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="schedGridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QTabWidget" name="schedTabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="delSchedPushButton">
       <property name="text">
        <string>Delete Schedule Sheet</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="addSchedPushButton">
       <property name="text">
        <string>Add Schedule Sheet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addSchedPushButton</tabstop>
  <tabstop>delSchedPushButton</tabstop>
  <tabstop>schedTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
